### ISSUE TITLE
cuckoofiles can be sabotaged in VM to disrupt analysis

### ISSUE THREAD

**Issue by kevross33** [cuckoosandbox/cuckoo#1029]
Here is an interesting one. Ransomware overwriting analyzer.py which seemed to upset the analyzer:

Error:
![analyzererror](https://cloud.githubusercontent.com/assets/2414517/17517687/6b089f74-5e3c-11e6-9b8a-51a583ea2390.PNG)

Ransomware Overwrite:
![analyzererror_overwrite](https://cloud.githubusercontent.com/assets/2414517/17517704/78f100ea-5e3c-11e6-9aa6-923b6ccd2fb4.PNG)

**Comment by jbremer** [comment]
Yes, you're right, however, the traceback you showed is a different issue - one that has been resolved recently. Please refer to #982. I'll keep this issue open to aim for hiding the Python and Cuckoo files etc.

**Comment by kevross33** [comment]
Ah right ok; I thought it had failed because the ransomware had encrypted
that file. Speaking of ransomware this is interesting and apparantly
implented on Cuckoo; hoping they release their prototype although I have
got a lot of milage with my generic sigs :-)

https://www.usenix.org/sites/default/files/conference/protected-files/security16_slides_kharaz_0.pdf
https://www.usenix.org/system/files/conference/usenixsecurity16/sec16_paper_kharraz.pdf

**Comment by jbremer** [comment]
Still on the todo list to merge everything :-) Thanks for the paper, will check it out as well.
